oonidata/netinfo.py

```diff
@@ -52,19 +52,21 @@ def load_databases(self):
         Where dat or mmdb indicates if the format is the legacy maxmind GeoIP or
         GeoIP2 formats respectively.
         """
-        for db_path in self.datadir.glob("*.gz"):
-            decompressed_path = db_path.with_suffix("")
+
+        # If the maxmind databases are compressed, we uncompress them.
+        for db_path in self.datadir.glob("*-country-asn.mmdb.gz"):
+            decompressed_path = db_path.with_suffix(".tmp")
             if decompressed_path.exists():
                 continue
             with gzip.open(db_path) as in_file, decompressed_path.open("wb") as out_file:
                 shutil.copyfileobj(in_file, out_file)
+            decompressed_path.rename(db_path.with_suffix(""))
 
         db_files = {}
         for db_path in self.datadir.glob("*"):
             if db_path.match("*.gz"):
                 continue
             ts = datetime.strptime(db_path.name.split("-")[0], "%Y%m%d").date()
-            db_files[ts] = db_files.get(ts, {})
```

Comment:
What was `ts`?

Reply:
This was a bug, that lead to not noticing if the netinfo db was able to load the database files. I'm not really sure how it came to be (I suspect I had in mind a different logic before, but forgot to remove it in here).

I noticed it when I was making improvements to the tests and had them run on an empty geoip datadir and instead of it failing early I was seeing it attempting to access using as key of a dict a dict.